packages/core/src/navigation/navigation-start.element.ts

```diff
@@ -120,36 +123,70 @@ export class CdsNavigationStart extends LitElement implements FocusableItem {
 
   private handleStartButtonText() {
     spanWrapper(this.childNodes);
+
     // get the projected text now wrapped in a span and add the sr attribute.
     this.querySelector('span')?.setAttribute(NAVIGATION_TEXT_WRAPPER, '');
   }
 
+  private renderRootStart() {
+    return html`
+      <slot @slotchange=${this.handleStartButtonText}></slot>
```

Comment:
For accessibility reasons when a navigation start is used ion the root, the text is projected outside of the button so that the button can act like a toggle and the text can remain independent from it.